site/app/models/Gradeable.php

```diff
@@ -485,6 +491,21 @@ protected function loadGradeableConfig() {
           $num_textboxes = count($details['textboxes']);
         }
 
+        if (isset($details['required_capabilities'])){
+            $this->required_capabilities = $details['required_capabilities'];
+        }
+        else{
+            $this->required_capabilities = "default";
+        }
+
+        if (isset($details['max_possible_grading_time'])){
+            $this->max_possible_grading_time = $details['max_possible_grading_time'];
+        }
+        else{
+           $this->max_possible_grading_time = -1;
+        }
```

Comment:
`$this->max_possible_grading_time = $details['max_possible_grading_time'] ?? -1;`